int16 IR
int8  mode
int16 realimu

uint8 sensorKiri
uint8 sensorKanan
int16 sensorUS1
int16 sensorUS2
int16 sensorUS3
